cypress/support/step-definitions/search-steps.js

```diff
@@ -18,7 +18,8 @@ When('Type {string} text into search input in no iFrame', (text) => {
 
 Then('Search component has input and {string} as icon', (iconType) => {
   searchInput().should('have.attr', 'data-element', 'input').and('be.visible');
-  searchInnerIcon().eq(FIRST_ELEMENT).should('have.attr', 'data-component', 'icon').and('be.visible')
+  searchInnerIcon().eq(FIRST_ELEMENT).should('have.attr', 'data-component', 'icon')
```

Comment:
could we use here also positionOfElement()?

Reply:
fixed